src/interfaces/parser.ts

```diff
@@ -43,6 +43,7 @@ export type Metadata = {
 
 type MetadataFlag = {
   setFromDefault?: boolean;
+  defaultHelp?: unknown;
```

Comment:
Need to retain value from defaultHelp to be used in later steps of parse/validation.